org.adempiere.base/src/org/compiere/model/MPInstanceLog.java

```diff
@@ -121,8 +165,18 @@ public String toString ()
 
 
 	private final static String insertSql = "INSERT INTO AD_PInstance_Log "
-			+ "(AD_PInstance_ID, Log_ID, P_Date, P_ID, P_Number, P_Msg, AD_Table_ID, Record_ID, AD_PInstance_Log_UU)"
-			+ " VALUES (?,?,?,?,?,?,?,?,?)";
+			+ "(AD_PInstance_ID, Log_ID, P_Date, P_ID, P_Number, P_Msg, AD_Table_ID, Record_ID, AD_PInstance_Log_UU, PInstanceLogType)"
+			+ " VALUES (?,?,?,?,?,?,?,?,?,?) "
+			+ " ON CONFLICT (AD_PInstance_Log_UU) "

```

Comment:
this is postgresql specific. please do the simple stuff here instead, separate insert and update sql

Reply:
I see, I haven't checked that, ok, fixed in the latest commit